CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project `3.1.0` per each release will be documented
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [v3.1.0.4]
+
+### Added
+
+- Introduce Helm chart for WSO2 APIM version 3.1.0 deployment pattern 4
+
```

Comment:
@chathuraswso2 please point to the relevant milestone (e.g. For detailed information on the tasks carried out during this release, please see the GitHub milestone [v3.1.0.3](https://github.com/wso2/kubernetes-apim/milestone/14)).

Reply:
Fixed in https://github.com/wso2/kubernetes-apim/pull/444/commits/678355f55e4758d7c45ac915b95fe88881616bc5